.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,106 @@
+name: GoReleaser
```

Comment:
Why not Nightly?

Reply:
Both versioned and nightly releases can be done using this workflow.

Reply:
Right, but I would not like to keep the action name as goreleaser in the future 👍

Reply:
I just saw the nightly flag on goreleaser which is great, prob just rename the action to Release